mmdet/utils/deployment/openvino_backend.py

```diff
@@ -242,3 +246,105 @@ def __call__(self, inputs, **kwargs):
             output['masks'] = output['masks'][valid_detections_mask]
 
         return output
+
+
+class MaskTextSpotterOpenVINO(ModelOpenVINO):
+    def __init__(self, *args, **kwargs):
+        self.with_mask = False
+        super().__init__(*args,
+                         required_inputs=('image', ),
+                         required_outputs=None,
+                         **kwargs)
+        self.n, self.c, self.h, self.w = self.net.input_info['image'].input_data.shape
+        assert self.n == 1, 'Only batch 1 is supported.'
+
+        xml_path = args[0].replace('.xml', '_text_recognition_head_encoder.xml')
```

Comment:
I guess you can pass the first argument in in an ordinary way.

Reply:
fixed

Reply:
fixed